index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <header id="header" >
+      <img id="header-img" src="Logo.png" alt="Logo">
+      <nav id="nav-bar"><a class="nav-link" href="#conocenos">Button 1</a> <a class="nav-link" href="#video">Button 2</a><a class="nav-link" href="#precios">Button 3</a></nav>
+    </header>
+    <div id="conocenos">
+      <form id="form" action="https://www.freecodecamp.com/email-submit" method="post">
+        <input id="email" type="email" placeholder="Enter your email" name="email" required>
+        <input id="submit" type="submit" value="Enviar">
+      </form>
+    </div>
+    <div>
+      <iframe id="video" width="500" height="500" src="https://encrypted-vtbn0.gstatic.com/video?q=tbn:ANd9GcSlHc71gi3CNZdDwB_sQFCqMWVrqZRhYVHPzw"></iframe>
```

Comment:
[OPTIONAL] As you are including materials from another source, I will suggest you include it inside a tag "article". Article is mostly used to separate parts of our project that have relevance by themselves or that are from information coming from another sourse.

Reply:
Hi, @HFG43!! 👋
- Well done on making inline comments with suggestions and reasoning. 👍 

- [ ] Kindly phrase the comments politely as suggestions instead of orders.
  - For example:
    - This is a polite suggestion: "I think it would be better to use `nav` nested of your `header`". 

- [ ] Kindly use emoticon (😄) so that your review looks human-friendly.

- [ ] ***[OPTIONAL]*** I can see some grammatical errors such as you have used, `I will suggest`, which would be `I  would suggest`. Kindly use [Grammarly](https://app.grammarly.com/) so that there are no spelling mistakes and other grammatical mistakes. It is more professional to check the spelling while giving any suggestion in every aspect of our job and profession.